samples/net/openthread/ncp/overlay-nrf-br.conf

```diff
@@ -0,0 +1,33 @@
+# Nordic BR
```

Comment:
I think this overlay name might be misleading, NCP over UART works on nRF out-of-the-box w/o any overlay (at least when I last tested it). We should include "usb" somewhere in the overlay name (for instance `overlay-usb-nrf-br.conf `) to clearly indicate it's for USB. 
Ideally, we should provide a generic USB overlay, that is not bound to any specific SoC/vendor, but I'm not sure if it's possible right now so I think it could stay like this.

Reply:
Renamed. Did not create a generic config yet.